### PR TITLE
added example notetook for titan multimodal embed model

### PR DESCRIPTION
**Pull request by kai-zhu-aws**
*Issue #, if available:*

*Description of changes:*
added example notetook for titan multimodal embed model

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by sunbc0120 — CHANGES_REQUESTED** [review]
Change notebook filename to "bedrock-titan-multimodal-embeddings.ipynb"

File: 05_Image/bedrock-titan-embed-image.ipynb [hunk 1]
```diff
@@ -0,0 +1,1226 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "801aa752-afe0-47bc-b73a-ec2667c9559a",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "def titan_multimodal_embedding(\n",
```
Comment (sunbc0120) [inline 05_Image/bedrock-titan-embed-image.ipynb:264]: refactor this lengthy function to:

```python
def titan_image_searching(
    image_path:str=None,  # maximum 2048 x 2048 pixels
    description:str=None, # English only and max input tokens 128
    dimension:int=1024,   # 1,024 (default), 384, 256
    model_id:str="amazon.titan-embed-image-v1"
):

    payload_body = {}
    embedding_config: dict = {
        "embeddingConfig": { 
             "outputEmbeddingLength": dimension
         }
    }

    # You can specify either text or image or both
    if image_path:
        # Maximum image size supported is 2048 x 2048 pixels
        with open(image_path, "rb") as image_file:
            input_image = base64.b64encode(image_file.read()).decode('utf8')
        payload_body["inputImage"] = input_image
    if description:
        payload_body["inputText"] = description

    assert payload_body, "please provide either an image and/or a text description"
    print("\n".join(payload_body.keys()))

    response = bedrock_client.invoke_model(
    	body=json.dumps({**payload_body, **embedding_config}), 
    	modelId=model_id,
    	accept="application/json", 
    	contentType="application/json"
    )

    return json.loads(response.get("body").read())
```

File: 05_Image/bedrock-titan-embed-image.ipynb [hunk 2]
```diff
@@ -0,0 +1,1226 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8d3eb696-b5b0-41e5-982b-8da18b9e3978",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "multimodal_embeddings = []\n",
```
Comment (sunbc0120) [inline 05_Image/bedrock-titan-embed-image.ipynb:324]: At this stage, let's:
- only index `image`
- Leave out `description.strip()`
- be explicit on `dimension=xxx`

File: 05_Image/bedrock-titan-embed-image.ipynb [hunk 3]
```diff
@@ -0,0 +1,1226 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4aece41e-40fa-46dc-8edc-f366f5d4136b",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "query_prompt = \"suede sneaker\"\n",
+    "query_emb = titan_multimodal_embedding(description=query_prompt)[\"embedding\"]\n",
```
Comment (sunbc0120) [inline 05_Image/bedrock-titan-embed-image.ipynb:441]: Be explicit on `dimension=XXX`

**Review by sunbc0120 — CHANGES_REQUESTED** [review]
Change notebook filename to "bedrock-titan-multimodal-embeddings.ipynb"

File: 05_Image/bedrock-titan-embed-image.ipynb (hunk 1 above)
Comment (kai-zhu-aws) [reply 05_Image/bedrock-titan-embed-image.ipynb:264]: done.

File: 05_Image/bedrock-titan-embed-image.ipynb (hunk 2 above)
Comment (kai-zhu-aws) [reply 05_Image/bedrock-titan-embed-image.ipynb:324]: done.

File: 05_Image/bedrock-titan-embed-image.ipynb (hunk 3 above)
Comment (kai-zhu-aws) [reply 05_Image/bedrock-titan-embed-image.ipynb:441]: done

**Review by sunbc0120 — APPROVED** [review]

**Pull request merged**